libs/langchain-google-webauth/src/tests/chat_models.int.test.ts

```diff
@@ -13,7 +13,7 @@ import { BaseLanguageModelInput } from "@langchain/core/language_models/base";
 import { ChatPromptValue } from "@langchain/core/prompt_values";
 import { ChatGoogle } from "../chat_models.js";
 
-describe("Google APIKey Chat", () => {
+describe.skip("Google APIKey Chat", () => {
```

Comment:
same

Reply:
These should probably stay and not be in the VertexAI classes since VertexAI doesn't support API Keys.

Reply:
Hmm now that I look again, there is no difference between the `Google APIKey Chat` tests and `Google WebAuth Chat` tests. I copy/pasted these from `@langchain/google-webauth`, is that a mistake from there?